subsys/net/l2/ethernet/ethernet.c

```diff
@@ -254,8 +254,13 @@ static enum net_verdict ethernet_recv(struct net_if *iface,
 		goto drop;
 #endif
 	default:
+#if defined(CONFIG_NET_ETHERNET_FORWARD_UNRECOGNISED_ETHERTYPE)
```

Comment:
Minor nit, we could actually do `if (IS_ENABLED(....))` here.

Reply:
Right, fixed.